modules/administration/pages/public-cloud-azure.adoc

```diff
@@ -0,0 +1,164 @@
+[[public-cloud-azure]]
+= {productname} with Azure
+
+You can use {productname} Server and Proxy with the Microsoft Azure public cloud.
+This section discusses what you will need for running {productname} in Azure, and how to set up your installation.
+
+
+== Configure the Azure Cloud Instance
+
+Use the ``SUSE Manager Server 4 BYOS`` image.
+The image is a pre-built image created by {suse}.
+It is based on JeoS, and SUSE Manager is pre-installed but not configured.
+Configuring SUSE Manager has to be done manually with {yast}.
+
+When you create your Azure virtual machine, choose the `B2ms` size.
+The `B2ms` VM contains:
+
+* 2{nbsp}vCPUs
+* 8{nbsp}GB RAM
+* Up to 4 data disks
+* Max IOPS 2400
+* Temporary storage disk of 16{nbsp}GB.
+Data on this disk will be destroyed after the guest has been switched off.
+
+When you are setting up disk partitioning, we recommend:
+
+* 30{nbsp}GB for the disk running the operating system
+* Select `Standard HDD` for the storage account type
+
+You will also require three additional data disks:
+
+* Disk 0: 64{nbsp}GB on Premium SSD, mounted at [path]``/var/lib/pgsql``
+* Disk 1: 512{nbsp}GB on Standard SSD, mounted at [path]``/var/spacewalk``
+* Disk 2: 128{nbsp}GB on Standard SSD, mounted at [path]``/var/cache``
+
+[NOTE]
+====
+Do not use LVM with Azure.
+If you need more disk space, extend a disk in the Azure portal, then extend the filesystem with [command]``xfs_growfs``.
+====
+
+Partition the disks like this:
+
+* [path]``/dev/sda``: 4 partitions containing the OS
+* [path]``/dev/sdb``: temporary storage disk, do not use
+* [path]``/dev/sdc``: contains [path]``/var/lib/pgsql``
+* [path]``/dev/sdd``: contains [path]``/var/spacewalk``
+* [path]``/dev/sde``: contains [path]``/var/cache``
+
+You can use these commands to create the disks:
+
+----
+for d in sdc sdd sde; do
+  parted --script /dev/$d mklabel gpt mkpart primary xfs 0% 100%
+  mkfs.xfs /dev/${d}1
+done
+mkdir /cachetmp
+mount /dev/sde1 /cachetmp
+cp -a /var/cache/* /cachetmp/
+umount /cachetmp
+echo "$(blkid /dev/sdc1|awk -F " " '{ print $2 }') /var/lib/pgsql xfs defaults,noatime 0 0" >> /etc/fstab
+echo "$(blkid /dev/sdd1|awk -F " " '{ print $2 }') /var/spacewalk xfs defaults,noatime 0 0" >> /etc/fstab
+echo "$(blkid /dev/sde1|awk -F " " '{ print $2 }') /var/cache     xfs defaults,noatime 0 0" >> /etc/fstab
+mkdir -p /var/spacewalk
+mount /var/spacewalk
+chown -R wwwrun:root /var/spacewalk
+mount /var/lib/pgsql
+chown -R postgres:postgres /var/lib/pgsql
+mv /var/cache /var/cache.old
+mkdir /var/cache
+mount /var/cache
+rm -r /var/cache.old
+----
+
```

Comment:
Any particular reason why the user shouldn't use the `suma-storage` script as described in the main public cloud page?

Reply:
We had some cases where we had performance problems. A deeper research showed that the root cause was having /var/spacewalk and /var/lib/pgsql on the same disk (could be different volumes). Also take in consideration that we normally but the backup of the database in the /var/spacewalk directory under backup-db. Postgress uses this also for storing the journals. So every operation against the database is written in the database and in the journal. When they are both on the same disk this could cause a bottleneck. But I have never tested/researched this with SSD disks. 
Another reason is that using both on the same disk increases the change of running out of diskspace eg when downloading new channels/products. And you don't want your database running out of diskspace....

Reply:
Safeguarding the data base from running out of disk space sounds definitely like a good idea. I guess suma-storage should be changed to create multiple partitions for that purpose on the storage volume, and possibly have support for multiple storage volumes added, in case somebody wants to set it up that way for performance reasons.

Regarding the potential performance issue, it's certainly valuable information, but I think it may require more research before we make specific recommendations in the documentation. As a side note, according to https://docs.microsoft.com/en-us/azure/virtual-machines/windows/disks-types, a small 64GB premium SSD volume doesn't really give you a better performance over a standard SSD volume of that size (maybe apart from a slightly lower latency). It'll cost more though.